routes/userRoute.js

```diff
@@ -0,0 +1,50 @@
+const
+_      = require('lodash'),
+User   = require('../models/User'),
+utils  = require('../modules/utils');
+
+function list(params, callback) {
+    callback({ message: 'not allowed' });
+}
+
+function get(params, callback) {
+    if (params.id) {
+        User.findOne({ _id: params.id }, utils.send(callback));
+    } else {
+        utils.send(callback);
```

Comment:
It will be called, but the `else` path will never be called, but I have the habit of writing `else` when I write `if` just in case I didn't think of all possibilities and that could result in a server timeout.